.zenodo.json

```diff
@@ -771,6 +780,9 @@
             "orcid": "0000-0001-8523-7223",
             "affiliation": "International Institute of Information Technology, Hyderabad"
         },
+        {
+            "name": "Hardik"
```

Comment:
@pythonicforge 

Would you like your name, affiliation and orcid added to the release?

Reply:
@pythonicforge , sorry for the additional ping.